manageiq-providers-kubernetes.gemspec

```
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency("hawkular-client",                 "~> 4.1")
   s.add_runtime_dependency("image-inspector-client",          "~> 2.0")
-  s.add_runtime_dependency("kubeclient",                      "~> 4.0")
+  s.add_runtime_dependency("kubeclient",                      "~> 4.1.1")
```

Minor, but if you want to keep the auto-upgrade on minor version, perhaps... 

```suggestion
  s.add_runtime_dependency("kubeclient",                      "~> 4.1", ">= 4.1.1")
```

---

Good catch.
I'm not aware of incompatible changes in any 4.y.
There is a second dependency through `fog-kubevirt`, currently on `kubeclient (~> 4.1.2)`, so it's better to not constrain without need — say later kubevirt discover they need something in 4.3, they won't need to bump two places.